pcdsdevices/happi/containers.py

```diff
@@ -665,3 +665,15 @@ class EnvironmentalMonitor(LCLSItem):
                        optional=True, enforce=str)
     ioc_base = EntryInfo('Base PV of the EK9000 IOC', optional=True,
                          enforce=str)
+
+
+from pcdsdevices.happi.containers import LCLSItem
+  
+class Leviton(LCLSItem):
+    kwargs = deepcopy(LCLSItem.kwargs)
+    kwargs.default['elevations'] = "{{elevations}}"
+    elevations = EntryInfo(doc = 'List of elevation numbers for rack',
```

Comment:
It took me a while to understand how you were able to cut down the class count so much. This elevations arg is a good solution to that :+1:.